cluster/ci/config/prow/config.yaml

```diff
@@ -299,6 +299,26 @@ presubmits:
     trigger: "((?m)^/test origin-verify,?(\\s+|$))"
 
   openshift/openshift-ansible:
+  - name: openshift/job/origin/job/build/
```

Comment:
Would be nice if the Jenkins logic could add in the `job/` segments itself when it's asked as the real name of the job is `openshift/origin/build`

Reply:
Hrm, not sure if there is a sane way to do this (how do we know that the name needs to be expanded? Just by the existence of slashes?) or whether we want to spend time fixing it at all.